packages/fakta-medlemskap/src/components/oppholdInntektOgPerioder/OppholdINorgeOgAdresserFaktaPanel.jsx

```diff
@@ -14,18 +15,27 @@ import { PeriodLabel, VerticalSpacer, FaktaGruppe } from '@fpsak-frontend/shared
 
 import styles from './oppholdINorgeOgAdresserFaktaPanel.less';
 
+countries.registerLocale(norwegianLocale);
+
 const capitalizeFirstLetter = landNavn => {
   const string = landNavn.toLowerCase();
   return string.charAt(0).toUpperCase() + string.slice(1);
 };
 
+const formatLandNavn = landNavn => {
+  if (landNavn.length === 2 || landNavn.length === 3) {
+    return countries.getName(landNavn, 'no');
+  }
+  return landNavn;
+};
+
 const lagOppholdIUtland = utlandsOpphold =>
   Array.isArray(utlandsOpphold) && utlandsOpphold.length > 0 ? (
     utlandsOpphold.map(u => (
       <div key={`${u.landNavn}${u.fom}${u.tom}`}>
         <Row>
           <Column xs="4">
-            <Normaltekst>{capitalizeFirstLetter(u.landNavn)}</Normaltekst>
+            <Normaltekst>{capitalizeFirstLetter(formatLandNavn(u.landNavn))}</Normaltekst>
```

Comment:
er det fortsatt behov for capitalizeFirstLetter?

Reply:
Ja, i tilfelle det man får inn ikke er en bokstavkode